isos/bootstrap/bootstrap

```diff
@@ -38,6 +38,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     mount -t tmpfs -o size=64m tmpfs ${MOUNTPOINT}/.tether/
 
     # enable full system functionality in the container
+    ln -s lib64 ${MOUNTPOINT}/.tether/lib
+    mkdir -p ${MOUNTPOINT}/.tether/{lib64,usr/lib/iptables}
```

Comment:
@mhagen-vmware It looks like `ln` will force `mkdir` if the dir does not exist. For example, the following succeeded on my local machine:
```
mkdir aaa
ln -s aaa bbb
ls -al | grep bbb
```
output: `bbb -> aaa`